specification/cognitiveservices/data-plane/Speech/SpeakerRecognition/identification/identification-swagger.json

```diff
@@ -0,0 +1,771 @@
+        },
+        "lastUpdatedDateTime": {
+          "$ref": "#/definitions/LastUpdatedDateTime"
+        },
+        "enrollmentsCount": {
+          "$ref": "#/definitions/EnrollmentsCount"
+        },
+        "enrollmentsLength": {
+          "$ref": "#/definitions/EnrollmentsLength"
+        },
+        "enrollmentsSpeechLength": {
+          "$ref": "#/definitions/EnrollmentsSpeechLength"
+        },
+        "remainingEnrollmentsSpeechLength": {
+          "$ref": "#/definitions/RemainingEnrollmentsSpeechLength"
+        },
+        "modelVersion": {
+          "$ref": "#/definitions/ModelVersion"
+        }
+      }
+    },
+    "EnrollmentInfo": {
+      "description": "Speaker profile enrollment info",
+      "properties": {
+        "profileId": {
+          "$ref": "#/definitions/ProfileId"
+        },
+        "enrollmentStatus": {
+          "$ref": "#/definitions/EnrollmentStatus"
+        },
+        "enrollmentsCount": {
+          "$ref": "#/definitions/EnrollmentsCount"
+        },
+        "enrollmentsLength": {
+          "$ref": "#/definitions/EnrollmentsLength"
+        },
+        "enrollmentsSpeechLength": {
+          "$ref": "#/definitions/EnrollmentsSpeechLength"
+        },
+        "remainingEnrollmentsSpeechLength": {
+          "$ref": "#/definitions/RemainingEnrollmentsSpeechLength"
+        },
+        "audioLength": {
+          "type": "number",
+          "description": "This enrolment audio length in seconds.",
+          "example": 1.83
+        },
+        "audioSpeechLength": {
+          "type": "number",
+          "description": "This enrollment audio pure speech (which is the amount of audio after removing silence and non-speech segments) length in seconds.",
+          "example": 1.35
+        }
+      }
+    },
+    "IdentifyInfo": {
+      "description": "Identified speaker info",
+      "properties": {
+        "profileId": {
+          "type": "string",
+          "pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$",
+          "description": "ID of identified of profile. If no candidate is identified as the right speaker, the value is set to empty GUID."
+        },
+        "score": {
+          "type": "number",
+          "description": "A float number indicating the similarity between input audio and targeted voice print. This number must be between 0 and 1. A higher number means higher similarity."
+        }
+      }
+    },
+    "Error": {
+      "required": [
+        "error"
+      ],
+      "properties": {
+        "error": {
+          "properties": {
+            "code": {
+              "type": "string"
+            },
+            "message": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    }
+  },
+  "responses": {
+    "NotFound": {
+      "description": "NotFound - Requested profile doesn't exist",
+      "schema": {
+        "$ref": "#/definitions/Error"
```

Comment:
@HeidiHanZhang 
The schema for your error as defined in your model does not match what you state you are returning in your examples. Specifically, you are missing the required "error" field. This should be fixed across all your examples as what you will actually return is a different structure.

Reply:
done